index.html

```diff
@@ -1,51 +1,236 @@
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- About Myself -->
+  <section class="about-me">
+    <div class="title">
+      <div class="img-content">
+        <h1>About me</h1>
+        <img src="./images/mask.svg" alt="" />
+      </div>
+      <p>
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="button-content">
+        <img src="./images/second-mask.svg" alt="button" />
+        <button>Get My Resume</button>
```

Comment:
**Optional**

- [ ] It will be nice to add the button type. This is because different browsers may use different default types for the `button` element.

Reply:
ok